templates/fine-tune-llm/job_compute_configs/gcp.yaml

```diff
@@ -1,8 +1,8 @@
 compute_config:
   allowed_azs:
     - any
-  cloud: my-cloud # You may specify `cloud_id` instead
-  region: any
+  cloud_id: cld_id # You may specify `cloud_id` instead
```

Comment:
hmm this is easy to miss. wonder if we should call out in readme.md that you need to set few required fields. Otherwise if people direct run with this file, they get a error with invalid cloud

Reply:
actual i see this mentioned in readme